base/client.jl

```diff
@@ -532,6 +532,10 @@ The thrown errors are collected in a stack of exceptions.
 """
 global err = nothing
 
+# For numbered REPL prompt
+global Out::Any
+export Out
+
```

Comment:
This doesn't seem like it should be here

Reply:
It reserves the binding. With me putting the old semantics back, it's probably no longer required, but may as well do it now.

Reply:
(Note that this is inside `MainInclude` like `ans` and `err`.

Reply:
IIUC, this is not supposed to have reserved the binding, as this binding actually is not supposed to be part of MainInclude originally, but rather is part of a plugin that sometimes adds this with this name for some alternative REPLs (but isn't necessarily the only name for it)

Reply:
Adding the binding at some random unspecified later time is just annoying though.

Reply:
That is sort of the point though, that although it is annoying to implement, but it is required to work correctly